server/php-engine.cpp

```diff
@@ -2106,6 +2115,8 @@ void parse_main_args(int argc, char *argv[]) {
   parse_option("disable-mysql-same-datacenter-check", no_argument, 2023, "Disable MySQL same datacenter check");
   parse_option("use-utf8", no_argument, 2024, "Use UTF8");
   parse_option("xgboost-model-path-experimental", required_argument, 2025, "intended for tests, don't use it for now!");
+  parse_option("statshouse-host", required_argument, 2026, "engine will connect to statshouse at this host and send stats to it");
+  parse_option("statshouse-port", required_argument, 2027, "engine will connect to statshouse at this port and send stats to it");
```

Comment:
let's don't call it 'engine', just server)

Reply:
merged this options and changed description